src/Components/CreateImageWizard/steps/Review/ReviewStep.tsx

```diff
@@ -334,7 +346,8 @@ const Review = ({ snapshottingEnabled }: { snapshottingEnabled: boolean }) => {
           <UsersList />
         </ExpandableSection>
       )}
-      {isTimezoneEnabled && (
+      {((isTimezoneEnabled && timezone) ||
+        (isTimezoneEnabled && ntpServers && ntpServers.length > 0)) && (
```

Comment:
This could be simplified  to checking for the flag as one condition and for any existing values as a second ANDed one. So like this:

```suggestion
      {isTimezoneEnabled &&
        (timezone || (ntpServers && ntpServers.length > 0)) && (
```